content/blog/posts/o3de-first-major-release-dec-2-2021.md

```diff
@@ -3,35 +3,33 @@ title: "Open 3D Foundation announces first major release of Open 3D Engine"
 description: "Simulation developers can now create 3D content with the new Linux editor and engine runtime, and a new Debian package and Windows installer provide a faster route to getting started with the engine."
 date: 2021-12-02
 slug: o3de-2111-announcement
-author: Liv Erickson
+author: Open 3D Foundation
 blog_img: "/images/blog/announcement_thumbnail.jpg"
 ---
 
-## Open 3D Foundation announces first major release of Open 3D Engine
-
 **The news in brief**: Simulation developers can now create 3D content with the new Open 3D Engine (O3DE) Linux editor and engine runtime, and a new Debian package and Windows installer provide a faster route to getting started with the engine.
 
-### O3DE's first major release: Stable Release 21.11
+### O3DE's first major release: Stable 21.11
 
-Today, the Open 3D Foundation (O3DF) and its members including Adobe, AWS, Huawei, Intel, Niantic, and RedHat announced the **O3DE Stable Release 21.11** of the Apache 2.0-licensed [Open 3D Engine](https://o3de.org/) project. Since the formation of O3DF and launch of the O3DE Developer Preview in July 2021, over 250 developers from a wide range of industries have contributed over 2,182 pull requests, 1,801 issues, and 8.56 million lines of code changes; developer features have also been added along with improved stability and performance to ensure that O3DE is ready for use in live games and simulations. In addition, O3DE now has a Windows installer to give users an accelerated onboarding experience as well as Linux support to bring O3DE to an even wider community of users and contributors. Developers can get started by downloading the installer from o3de.org or cloning the source repository from GitHub.
+Today, the Open 3D Foundation (O3DF) and its members including Adobe, AWS, Huawei, Intel, and Niantic announced the **O3DE Stable 21.11 release** of the Apache 2.0-licensed [Open 3D Engine](https://o3de.org/) project. Since the formation of O3DF and launch of the O3DE Developer Preview in July 2021, over 250 developers from a wide range of industries have contributed over 2,182 pull requests and 1,801 issue; developer features have also been added along with improved stability and performance to ensure that O3DE is ready for use in live games and simulations. In addition, O3DE now has a Windows installer to give users an accelerated onboarding experience as well as Linux support to bring O3DE to an even wider community of users and contributors. Developers can get started by downloading the installer from o3debinaries.org or cloning the source repository from GitHub.
```

Comment:
```suggestion
Today, the Open 3D Foundation (O3DF) and its members including Adobe, AWS, Huawei, Intel, and Niantic announced the **O3DE Stable 21.11 release** of the Apache 2.0-licensed [Open 3D Engine](https://o3de.org/) project. Since the formation of O3DF and launch of the O3DE Developer Preview in July 2021, over 250 developers from a wide range of industries have contributed over 2,182 pull requests and 1,801 issues; developer features have also been added along with improved stability and performance to ensure that O3DE is ready for use in live games and simulations. In addition, O3DE now has a Windows installer to give users an accelerated onboarding experience as well as Linux support to bring O3DE to an even wider community of users and contributors. Developers can get started by downloading the installer from o3debinaries.org or cloning the source repository from GitHub.
```